data/json/monsters/nether.json

```
@@ -875,5 +875,56 @@
     "harvest": "exempt",
     "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s melts away." },
     "flags": [ "HEARS", "GOODHEARING", "STUMBLES", "NOHEAD", "HARDTOSHOOT", "FLIES", "PLASTIC", "NO_BREATHE", "NOGIB" ]
+  },
+  {
+    "id": "mon_balrog",
+    "type": "MONSTER",
+    "name": { "str": "fiery nightmare" },
+    "description": "A colossal figure which seems to consist entirely of shadows engulfed in flames.  Its immense dark wings extend from wall to wall.  Two eyes that resemble burning coals are gazing down on you with hatred.  In one hand it holds a huge sword, in the other it has a many thonged whip, both glowing with red-hot flames.",
+    "default_faction": "nether",
+    "bodytype": "human",
+    "species": [ "UNKNOWN" ],
+    "volume": "875000 ml",
+    "weight": "300 kg",
+    "hp": 500,
+    "speed": 140,
+    "symbol": "B",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 10,
+    "melee_dice": 7,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "heat", "amount": 20 } ],
+    "melee_cut": 10,
+    "armor_bash": 12,
+    "armor_cut": 12,
+    "armor_bullet": 12,
```

```suggestion
    "armor_bullet": 25,
```
I think animate shadows & flames should ignore more damage from small projectiles comparatively to what they take from cut and bash.